src/browser/components/SkillItem.react.js

```diff
@@ -31,15 +31,29 @@ export default class SkillItem extends Component {
     const STARS_LENGTH = 5;
     for (let i = STARS_LENGTH; i >= 1; i--) {
       starsLabels.push(
-          <input className={`star star-${i}`} id={`star${i}_${this.props.data.id}`} key={i} name="star" onChange={(e) => this.handleStarChange(i, e)} type="radio"/>,
-          <label className={`star star-${i}`} htmlFor={`star${i}_${this.props.data.id}`} key={i + STARS_LENGTH} />
+          <input className={`star star-${i}`}
+                 id={`star${i}_${this.props.data.id}`}
+                 key={i}
+                 name="star"
+                 onChange={(e) => this.handleStarChange(i, e)}
+                 type="radio"/>,
+
+          <label className={`star star-${i}`}
+                 htmlFor={`star${i}_${this.props.data.id}`}
+                 key={i + STARS_LENGTH} />
       );
     }
 
     return (
       <li>
-        {this.state.hintDisplayed ? <div className="hint">{this.props.data.name}</div> : false}
-        <img alt={this.props.data.name} onMouseEnter={()=>this.showHint()} onMouseLeave={()=>this.hideHint()} src={this.props.data.src}/>
+        {this.state.hintDisplayed ?
+          <div className="hint">{this.props.data.name}</div> : false}
+
+        <img alt={this.props.data.name}
+             onMouseEnter={()=>this.showHint()}
+             onMouseLeave={()=>this.hideHint()}
```

Comment:
Unneeded arrow function here. I think `this.showHint` and `this.hideHint` would be enough